lib/domain/usecase/send_file_interactor.dart

```diff
@@ -18,13 +18,25 @@ class SendFileInteractor {
         final txid = room.storePlaceholderFileInMem(
           fileInfo: fileInfo,
         );
-        room.sendFileEvent(
+
+        final fakeImageEvent = await room.sendFakeImagePickerFileEvent(
```

Comment:
`fakeImageEvent` -> `fakeFileEvent`

Reply:
Not use remove `send_file_interactor.dart`